wagtail/admin/ui/tables/__init__.py

```diff
@@ -421,6 +424,7 @@ def get_row_classname(self, instance):
 
     def get_row_attrs(self, instance):
         attrs = {}
+        attrs["data-w-sortable-target"] = self.sortable_item_attr
```

Comment:
We should not need changes to this file at all if we make the `get_row_attrs` change inside `wagtail/admin/ui/tables/pages.py`. There is already logic to add the row attributes based on the `self.use_row_ordering_attributes`.